README.md

```diff
@@ -28,4 +28,107 @@ App that allow customers to collaborate being moved around the city by either cr
     You can check that your server is now working in a browser:
         `http://127.0.0.1:8000/api/accounts/`
     Press `Ctrl+C` when completed to stop the services.
+    
+ ## Description of API endpoints
```

Comment:
Its better to put all this comments under docstring directly in views. But you should filter out some information out of here ( F.E. information about start_point and dest_point.) This kind of information should be placed directly under func or class which operates with information..

Reply:
You should understand that almost all people who would read your code - developers. And there is no need to explain some obvious things that man can understand from signature of a function. Especially if function/class has good/brief documentation about what it does.